plaidml/edsl/__init__.py

```diff
@@ -774,19 +774,30 @@ class SortDirection(enum.IntEnum):
     """Sort a tensor in descending order."""
 
 
-def argsort(x, axis=-1, direction=SortDirection.ASC):
+class SortStability(enum.IntEnum):
+    """Stability for a sort operation."""
+
+    STABLE = 0
+    """Sort a tensor in a stable way."""
+
+    UNSTABLE = 1
+    """Sort a tensor in an unstable way."""
+
+
+def argsort(x, axis=-1, direction=SortDirection.ASC, stability=SortStability.UNSTABLE):
     """Returns the indices of a tensor ``x`` that give its sorted order along
     an ``axis``, where -1 represents the last axis, in either ascending or
-    descending ``direction``.
+    descending ``direction`` and with either stable or unstalbe ``stability``.
 
     Args:
         x (Tensor): The tensor to be sorted.
         axis (TensorDim): The axis to perform sorting on.
         direction (SortDirection): The direction of value comparison.
+        stability (SortStability): The stability of value comparison.
```

Comment:
Maybe say "Whether the sort is stable" instead? Stability is more about whether certain reorderings are legal than about how comparisons are made.